galleries/users_explain/text/mathtext.py

```diff
@@ -6,366 +6,364 @@
-
-.. TODO: Link to font chapter
-
-The fonts used should have a Unicode mapping in order to find any
-non-Latin characters, such as Greek.  If you want to use a math symbol
-that is not contained in your custom fonts, you can set
-:rc:`mathtext.fallback` to either ``'cm'``, ``'stix'`` or ``'stixsans'``
-which will cause the mathtext system to use
-characters from an alternative font whenever a particular
-character cannot be found in the custom font.
-
-Note that the math glyphs specified in Unicode have evolved over time, and many
-fonts may not have glyphs in the correct place for mathtext.
-
-Accents
--------
-An accent command may precede any symbol to add an accent above it.  There are
-long and short forms for some of them.
-
-============================== =================================
-Command                        Result
-============================== =================================
-``\acute a`` or ``\'a``        :mathmpl:`\acute a`
-``\bar a``                     :mathmpl:`\bar a`
-``\breve a``                   :mathmpl:`\breve a`
-``\dot a`` or ``\.a``          :mathmpl:`\dot a`
-``\ddot a`` or ``\''a``        :mathmpl:`\ddot a`
-``\dddot a``                   :mathmpl:`\dddot a`
-``\ddddot a``                  :mathmpl:`\ddddot a`
-``\grave a`` or ``\`a``        :mathmpl:`\grave a`
-``\hat a`` or ``\^a``          :mathmpl:`\hat a`
-``\tilde a`` or ``\~a``        :mathmpl:`\tilde a`
-``\vec a``                     :mathmpl:`\vec a`
-``\overline{abc}``             :mathmpl:`\overline{abc}`
-============================== =================================
-
-In addition, there are two special accents that automatically adjust to the
-width of the symbols below:
-
-============================== =================================
-Command                        Result
-============================== =================================
-``\widehat{xyz}``              :mathmpl:`\widehat{xyz}`
-``\widetilde{xyz}``            :mathmpl:`\widetilde{xyz}`
-============================== =================================
-
-Care should be taken when putting accents on lower-case i's and j's.  Note that
-in the following ``\imath`` is used to avoid the extra dot over the i::
-
-    r"$\hat i\ \ \hat \imath$"
-
-.. math::
-
-    \hat i\ \ \hat \imath
-
-Symbols
--------
-You can also use a large number of the TeX symbols, as in ``\infty``,
-``\leftarrow``, ``\sum``, ``\int``.
-
-.. math_symbol_table::
-
-If a particular symbol does not have a name (as is true of many of the more
-obscure symbols in the STIX fonts), Unicode characters can also be used::
-
-   r'$\u23ce$'
-
-Example
--------
-Here is an example illustrating many of these features in context.
-
-.. figure:: /gallery/text_labels_and_annotations/images/sphx_glr_mathtext_demo_001.png
-   :target: /gallery/text_labels_and_annotations/mathtext_demo.html
-   :align: center
 """
+
+# %%
+# Any text element can use math text.  You should use raw strings (precede the
```

Comment:
Raw strings are not required; they are just more convenient since you don't have to escape the backslashes which occur more often for TeX.